validator/src/main/java/com/amazon/aoc/validators/ContainerInsightStructuredLogValidator.java

```diff
@@ -20,17 +20,10 @@
         extends AbstractStructuredLogValidator {
 
   private static final List<String> LOG_TYPE_TO_VALIDATE = Arrays.asList(
-          "Cluster",
```

Comment:
The log for eks and ecs container insight is different, I am not sure if you need to update this file since you already defined the same list in `ConatinerInsightECSStructuredLogValidator.java`

Reply:
Thanks for pointing it out, I forget correcting it.